packages/terra-props-table/src/PropsTable.jsx.orig

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Not sure whats up with this file, looks like this wasn't meant to be checked into version control

Reply:
Whoops! looks like that was left over from my diff, f85e7b2c